_data/roadmap.yml

```diff
@@ -1,60 +1,60 @@
 - year: 2014
   accomplishments:
     - name: Launched on Bitcointalk
-      date: April 18, 2014
+      date: 18 April 2014
```

Comment:
I would make a case for YYYY-MM-DD even though it's second most common in the world by population... It's also used as secondary format in many countries next to the most popular and it leaves no room for confusion.